modules/lightning_features/lightning_media/lightning_media.module

```diff
@@ -22,9 +22,18 @@ function lightning_media_form_entity_browser_media_browser_form_alter(array &$fo
  * Implements hook_entity_type_alter().
  */
 function lightning_media_entity_type_alter(array &$entity_types) {
-  // Files should never be workspace-aware. Besides, they ALWAYS conflict during
-  // replication, for reasons as yet unknown.
-  $entity_types['file']->set('multiversion', FALSE);
+  if (\Drupal::moduleHandler()->moduleExists('multiversion')) {
+    // Files should never be workspace-aware. Besides, they ALWAYS conflict during
+    // replication, for reasons as yet unknown. But because Multiversion is
+    // basically a one-way ticket, only opt file entities out of Multiversion if
+    // they haven't been migrated into Multiversion's storage yet. There really
```

Comment:
Should this have it's own issue? I'm trying to understand when a file entity would not have yet been migrated to multiversion's storage. I know from anecdotal experience that files created after installing multiversion so still conflict, so I'm wondering if this would even address that problem.